smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -472,8 +472,6 @@ def addBridge() {
 					log.error "Failed to create Hue Bridge device"
 				}
 			}
-		} else {
-			log.debug "found ${d.displayName} with id $selectedHue already exists"
```

Comment:
just didnt want it to print this every single poll for every user since it is the normal case.